WorkloadManagementSystem/Utilities/JobMemory.py

```diff
@@ -1,9 +1,9 @@
 """ DIRAC Workload Management System Client module to get available memory from mjf
 """
-__RCSID__ = "$Id$"
-
 import os
-import urllib
+import requests
```

Comment:
Thanks for letting me know this module ;-)
We certainly have quite a few places where this would be useful (in MJF for example)